src/test/java/org/folio/rest/support/builders/LoanRequestBuilder.java

```diff
@@ -10,22 +10,23 @@
 
 public class LoanRequestBuilder implements Builder {
 
-  private final UUID id;
-  private final UUID itemId;
-  private final UUID userId;
-  private final UUID proxyUserId;
-  private final DateTime loanDate;
-  private final String statusName;
-  private final String itemStatus;
-  private final DateTime dueDate;
-  private final String action;
-  private final String actionComment;
-  private final UUID itemEffectiveLocationAtCheckOut;
-  private final UUID loanPolicyId;
+  private UUID id;
+  private UUID itemId;
+  private UUID userId;
+  private UUID proxyUserId;
+  private DateTime loanDate;
+  private String statusName;
+  private String itemStatus;
+  private DateTime dueDate;
+  private String action;
+  private String actionComment;
+  private UUID itemEffectiveLocationAtCheckOut;
+  private UUID loanPolicyId;
```

Comment:
Why have these properties been made non-final / mutable?

Reply:
It is so cumbersome to add new fields(the need to update all the constructors, just to add one field?).  I could not find any justifications of bloating the constructor even more.

Reply:
The intent was that any instance of the builder is immutable, meaning it can be used as a prototype / template, so can be shared or used as a base line for multiple operations.

Making it mutable means that all instances of the builder based upon the same chain will also get that change.

I don't really follow the argument that this is just a test class. However I'm not inclined to argue if you are intent on making this change.

Reply:
Done.